js/palette.js

```diff
@@ -69,3 +70,16 @@ let palette_color_array = [
     "#000001",
     "#000000",
 ];
+
+//Classic Gameboy and Gameboy pocket color palettes
```

Comment:
Awesome, thanks for keeping the style consistent across this file with comment and grouped colors